### ISSUE TITLE
When using delegate, enable()/disable() do not affect tooltips

### ISSUE THREAD

**Issue by FelixRilling** [atomiks/tippyjs#843]
## Bug description

When using delegate, enable()/disable() do not affect tooltips.
Unlike 'normal' tippy instances, delegate instances seem to ignore calls to enable and disable them.

## Reproduction

https://codepen.io/FelixRilling/pen/mdVoaGW?editors=1010

**Comment by owenfar** [comment]
Hi @FelixRilling, have you found a workaround for this? I get the instance successfully but when trying both disable/hide methods on a delate instance, nothing seems to work.

**Comment by FelixRilling** [comment]
@owenfar I don't have a workaround, but I think i spotted why it does not work int the code:
in `delegate` , `applyMutations` re-binds #destroy to a function which calls the method on all children first - this might also need to be done for enable/disable.

**Comment by owenfar** [comment]
@FelixRilling That's great, thanks for letting me know. For now I'll hack another way to hide the tooltips while dragging. Good luck with the fix, and hope to see it patched soon :)

**Comment by FelixRilling** [comment]
I started working on a fix in https://github.com/FelixRilling/tippyjs/tree/delegate-all, so far I think its an OK fix. One issue I'm having right now I would want to hear one of the maintainers (@atomiks ) opinions on the handling disabling/hiding directly after creation:

while programatically setting new instances to disabled works, the tooltip is still visible for a bit if `showOnCreate` is set. My idea was overriding the `showOnCreate` from the nativeProps props to be false, and then later set the correct visibility manually. However I saw that currently all children are forced to have `showOnCreate` to be true (` {...nativeProps, showOnCreate: true};`), and I wanted to ask if its OK to override that conditionally.
